frontend/src/pages/kakaoCallback/KakaoCallback.tsx

```diff
@@ -12,50 +12,49 @@ import { postKakaoLogin } from './apis/postKakaoLogin';
 function KakaoCallback() {
   const navigate = useNavigate();
   const { login } = useAuth();
+  const didLoginRef = useRef(false);
 
   useEffect(() => {
     const authCode = new URLSearchParams(window.location.search).get('code');
 
-    if (authCode) {
-      const handleLogin = async (authCode: string) => {
-        try {
-          const response = await postKakaoLogin(authCode);
-
-          if (response.status === 200) {
-            login();
-            navigate(PAGE_URL.HOME);
-          } else if (response.status === 204) {
-            navigate(PAGE_URL.IDENTITY_VERIFICATION);
-          } else {
-            captureSentryError({
-              error: new Error(
-                `Unexpected response status: ${response.status}`,
-              ),
-              level: 'warning',
-              feature: 'auth',
-              step: 'kakao-login',
-            });
-            alert('로그인에 실패했습니다.');
-            navigate(PAGE_URL.LOGIN);
-          }
-        } catch (error) {
-          console.error('카카오 로그인 에러', error);
+    if (!authCode || didLoginRef.current) return;
+    didLoginRef.current = true;
+
+    const handleLogin = async (authCode: string) => {
+      try {
+        const response = await postKakaoLogin(authCode);
+
+        if (response.status === 200) {
+          login();
+          window.history.replaceState(null, '', PAGE_URL.HOME);
+          navigate(PAGE_URL.HOME);
```

Comment:
p4: window.history.replaceState 대신 navigate에서 제공하는 값을 사용해도 좋을 거 같아! 👍

```suggestion
          navigate(PAGE_URL.HOME, {replace: true});
```

Reply:
[refactor: window.history.replaceState 대신 navigate 사용](https://github.com/woowacourse-teams/2025-Fit-toring/pull/891/commits/4a0b811da6190fae502201b839df085aae7f44d0)